tmt/steps/provision/podman.py

```diff
@@ -118,6 +118,32 @@ def pull_image(self) -> None:
             message=f"Pull image '{self.image}'."
             )
 
+    def _setup_network(self) -> list[str]:
+        """
+        Set up the desired network.
+        Will look for existing network using the tmt workdir name,
+        or will create that network if it doesn't exist.
+        Returns the network arguments to be used in podman run command.
+        """
+
+        run_id = self._tmt_name().split('-')[1]
+        self.network = f"tmt-{run_id}-network"
+
+        try:
+            self.podman(
+                Command('network', 'create', self.network),
+                message=f"Creating network '{self.network}'."
+                )
+        except tmt.utils.RunError as err:
+            if 'network already exists' in err.message:
```

Comment:
What if I use other locale than EN? But this can be addressed in separate pr, I see we have at least two other checks for error msg

Reply:
Looking at the source code for podman, as well doing some local testing ->  podman does not seem support other LANG than EN?  There are some translation files in JP for man pages but that's it...